docs/user_workloads/creating_it_pref.md

```diff
@@ -0,0 +1,72 @@
+# Creating Instance Types
+
+The virtctl subcommand `create instancetype` allows easy creation of an instance type
+manifest from the command line. The command also provides several flags that can be used
+to create your desired manifest.
+
+There are two required flags that need to be specified: the number of vCPUs and the amount of
+memory to be requested. Additionally, there are several optional flags that can be used, such as specifying
+a list of GPUs for passthrough, choosing the desired IOThreadsPolicy, or simply providing
+the name of our instance type.
+
+By default, the command creates the cluster-wide resource. If the user wants to create the namespaced version,
+they need to provide the namespaced flag. The namespace name can be specified by using the namespace flag.
```

Comment:
Could you please specify the flag? E.g.
"The namespace name can be specified by using the flag `--namespaced"